src/sentry/conf/server.py

```diff
@@ -1250,6 +1250,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:sentry-functions": False,
     # Enable experimental session replay backend APIs
     "organizations:session-replay": False,
+    # Enable Session Replay showing in the sidebar
+    "organizations:session-replay-ui": True,
```

Comment:
In the cases where it needs to be disabled, will they have this already set to False?

Reply:
yes, https://github.com/getsentry/ops/pull/6274/files#diff-af45c621dda644c109b08805c6e96f9a877495c27dd1ccd649fad7648fb86f4eR511